bundles/org.openhab.core.io.rest/src/main/java/org/openhab/core/io/rest/JSONResponse.java

```diff
@@ -173,7 +173,7 @@ private Response createResponse(Response.StatusType status, final Object entity)
                 gson.toJson(entity, entity.getClass(), jsonWriter);
                 jsonWriter.flush();
             } catch (IOException | JsonIOException e) {
-                logger.error("Error streaming JSON through PipedInpuStream/PipedOutputStream: ", e);
```

Comment:
I agree these should be logged on debug if they are expected due to connection loss. It would be nice if there was also some client ID in the message but it may not be easy to add it at this level.

Can you remove the ": " at the end of the log messages? They suggest there should is some log detail but it is not there. It also suggests exceptions are logged in a certain way which may not always be the case. E.g. you can configure logging so it doesn't append stacktraces at all or you can append logging to a database or HTML (with fancy tables). In those cases the stacktrace may not immediately follow the log message.

Reply:
Done.